### PR TITLE
Update 2017-11-03-nearing-the-7.0-release.md

### PR DESCRIPTION
**Pull request by jonathantneal**
I made a few stylistic changes based on what flowed best for me.

- Use less contractions (but preserve the negative contractions because I like them).
- Group paragraphs by thought (because I can read your mind?).
- Preserve a tense per paragraph (past tense, present tense, future tense, camping tents).
- Preserve author perspective (first person "I", third person "We", bad person "Judas Iscariot").
- Pretend I know where commas go.

---

Also, I’m not sure what this section is referring to:

> `babel-preset` is actually a pretty *old* preset that replaces every other syntax preset that you will need (es2015, es2016, es2017, es20xx, latest, etc...).

Is `babel-preset` what you meant? Or `babel-preset-env`? I can fix that, too.

### CODE REVIEW

File: _posts/2017-11-03-nearing-the-7.0-release.md [hunk 1]
```diff
@@ -8,25 +8,26 @@ categories: announcements
 share_text: "Nearing the 7.0 Release"
 ---
 
-> Please check out the previous [Planning for 7.0](https://babeljs.io/blog/2017/09/12/planning-for-7.0) post for information up till the first beta release!
+> [Hey! Listen!](https://soundcloud.com/omnomthenom/hey-listen-a-dubstep-tribute-to-navi) Check out [Planning for 7.0](https://babeljs.io/blog/2017/09/12/planning-for-7.0) for updates throughout the beta releases.
```
[the remark this answers is not in the document]
Comment (xtuc) [reply _posts/2017-11-03-nearing-the-7.0-release.md]: Btw we did also alpha releases, not only beta but that doesn't really matter here.

File: _posts/2017-11-03-nearing-the-7.0-release.md [hunk 2]
```diff
@@ -8,25 +8,26 @@ categories: announcements
...
 [![videos](https://i.imgur.com/DkBEsfo.png)](https://babeljs.io/docs/community/videos/)
 
-- We created a new [team page](https://babeljs.io/team)! We'll be updating this in the future with more information about what people work on and why they are involved! For such a large project, there are many ways to get involved and help out!
+- We created a new [team page](https://babeljs.io/team)! We will be updating this page in the future with more information about what people work on and why they are involved. For such a large project, there are many ways to get involved and help out.
 
 [![team](https://i.imgur.com/YcWgRwf.png)](https://babeljs.io/team)
 
-- Babel turned three years old on [Sept 28](https://babeljs.io/blog/2017/10/05/babel-turns-three)!
-- Daniel [moved](https://twitter.com/left_pad/status/926096965565370369) `babel/babylon` and `babel/babel-preset-env` into the main Babel monorepo: [babel/babel](https://github.com/babel/babel), including everything from git history, labels, issues!
+- Babel turned 3 years old on [September 28, 2017](https://babeljs.io/blog/2017/10/05/babel-turns-three)!
```
Comment (xtuc) [inline _posts/2017-11-03-nearing-the-7.0-release.md]: Why three -> 3? In french you write down the number < 10. Is that the same in english?

File: _posts/2017-11-03-nearing-the-7.0-release.md [hunk 3]
```diff
@@ -64,46 +65,46 @@ We will also be working on a upgrade tool that will help [rewrite your package.j
...
 
 We can always keep it up to date, and maybe we just need to determine a better system than what these presets are currently.
 
-Right now, they are literally just a list of plugins that we manually update after a TC39 meeting happens. Of course to make this managable we will need to just allow for major version bumps for these "unstable" packages. Part of the reason for this is because the community will just re-create these packages anyway, so we might as well do it via an official package and have the ability to provide better messaging, etc.
+Right now, stage presets are literally just a list of plugins that we manually update after TC39 meetings. To make this managable, we need to allow major version bumps for these "unstable" packages. Part of the reason for this is because the community will re-create these packages anyway, so we might as well do it from an official package and then have the ability to provide better messaging, etc.
 
 ### Renames: Scoped Packages (`@babel/x`)
 
-Alright we only just changed every single package for Babel again 😂..
+Alright, we only just changed every single package for Babel again 😂..
```
Comment (xtuc) [inline _posts/2017-11-03-nearing-the-7.0-release.md]: I would prefer to not include this sentence at all. We don't need to say that "we broke everything for you once again".

File: _posts/2017-11-03-nearing-the-7.0-release.md [hunk 4]
```diff
@@ -64,46 +65,46 @@ We will also be working on a upgrade tool that will help [rewrite your package.j
...
 
-Back then, there weren't a lot of projects using scoped packages so most people didn't even know they existed. I believe you also had to pay for a npm org account while now it's free (and supports non scoped packages too). The issues with searching for scoped packages are solved and download counts work. The only thing left is that some 3rd party registries still don't support scoped. I think we are at a point where it seems pretty unreasonable to wait on that.
+Back then, not a lot of projects used scoped packages, so most people didn't even know they existed. You may also have had to pay for a npm org account back then, while now it's free (and supports non scoped packages, too). The issues with searching for scoped packages are solved and download counts work. The only thing left is that some 3rd party registries still don't support scoped packages, and I think we are at a point where it seems pretty unreasonable to wait on that.
 
-If you want reasons why:
+If you want reasons why I prefer scoped packages:
```
Comment (xtuc) [inline _posts/2017-11-03-nearing-the-7.0-release.md]: why I prefer scoped packages -> why we prefer scoped packages

File: _posts/2017-11-03-nearing-the-7.0-release.md (hunk 3 above)
Comment (hzoo) [reply _posts/2017-11-03-nearing-the-7.0-release.md]: It's not broken though? renaming a package doesn't break the existing package, but sure we can remove it if you want

Comment (xtuc) [reply _posts/2017-11-03-nearing-the-7.0-release.md]: No it's not but I mean that's what I understand when I read this.

**Comment by jonathantneal** [comment]
🎉

**Pull request merged**